### ISSUE TITLE
Unavailable image link needs alt text

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#15235]
**Describe the bug**
AMP detects missing accessible name for unavailable image, since this is not a real image, the link itself needs text. I suggest adding `aria-label` to the link.

![Screen Shot 2022-02-23 at 8 19 31 PM](https://user-images.githubusercontent.com/15113219/155440470-ad2c2fb1-3f4f-45fd-a7cf-4f8795fe0d70.png)

**Expected behaviour**
We have two options, please implement whichever is possible.

- **Preferred** solution: SR says vocalizes: _"no image available, product name"_ 

![Screen Shot 2022-02-23 at 8 27 10 PM](https://user-images.githubusercontent.com/15113219/155440865-caf639ec-8901-4095-bcab-70d8d1e808c1.png)

- **Generic** solution: SR vocalizes: _"no image available"_ for all unavailable images.

![Screen Shot 2022-02-23 at 8 24 22 PM](https://user-images.githubusercontent.com/15113219/155440886-803e4ca1-0cd8-4843-877c-5ebb97944768.png)

**Comment by dydome** [comment]
@developpeurweb PR is ready to review but I'm not sure if we need this? A tag has `tabindex` set to `-1` so we are not able to tab on it, so how SR will read it? https://github.com/SAP/spartacus/pull/15270

**Comment by developpeurweb** [comment]
@dydome the non-focusable link got fixed on GH-15215 pls coordinate with @pawelfras so your PRs won't overlap.
